common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1026,13 +1027,13 @@
     "figura.docs.host.get_status_effects": "Returns a table of all of the player's status effects\nThe table contains sub-tables, each of which contains the name, amplifier, duration, and particle visibility of each status effect",
     "figura.docs.host.get_clipboard": "Gets the text from the clipboard",
     "figura.docs.host.set_clipboard": "Sets the clipboard text",
-    "figura.docs.host.get_attack_charge": "Returns a fraction (0 to 1) of the charge of the player attack\nIf less than 1, every attack will result result in a weak attack",
+    "figura.docs.host.get_attack_charge": "Returns a fraction (0 to 1) of the charge of the player attack\nIf less than 1, every attack will result in a weak attack",
     "figura.docs.host.is_jumping": "Checks if the player is jumping\nNote this is only true during the first tick the player started jumping",
     "figura.docs.host.is_flying": "Checks if the player is currently flying",
     "figura.docs.host.get_reach_distance": "Returns the current reach distance of the player",
     "figura.docs.host.get_air": "Gets the remaining amount of air of the player",
-    "figura.docs.host.get_pick_block": "Returns the current targeted block set by the client\nReturns a vararg of the block, the hit position and the block face the hit collided",
-    "figura.docs.host.get_pick_entity": "Returns the current targeted entity set by the client",
+    "figura.docs.host.get_pick_block": "Returns the current targeted block set by the client\nReturns as multiple results of the block, the hit position and the block face the hit collided",
```

Pretty much copied from `figura.docs.entity.get_targeted_block`.
```suggestion
    "figura.docs.host.get_pick_block": "Returns the current targeted block set by the client\nReturns the block, the hit position, and the targeted block face as three separate values",
```